lib/rock/webapp/tasks/api.rb

```diff
@@ -102,6 +102,55 @@ def get_operation(name_service, name, operation_name)
                         Hash[task: task_by_name(params[:name_service], params[:name]).to_h]
                     end
                     
+                    desc "starts a task"
+                    get ':name_service/:name/start' do
+                        task = task_by_name(params[:name_service], params[:name])
```

Comment:
With the test on rtt_state gone (see below), the whole code can be factored out. Replace all of these by a state_transition endpoint which takes the command (start, stop, cleanup, ...) as option.